src/AutoNumeric.js

```diff
@@ -7413,7 +7413,7 @@ To solve that, you'd need to either set \`decimalPlacesRawValue\` to \`null\`, o
         this.isEditing = false;
 
         // Send a `change` event if the raw value has been changed since the last focus or 'enter' validation
-        if (this.rawValue !== this.rawValueOnFocus) {
+        if (AutoNumericHelper.trimPaddedZerosFromDecimalPlaces(this.rawValue) !== AutoNumericHelper.trimPaddedZerosFromDecimalPlaces(this.rawValueOnFocus)) {
```

Comment:
I'm not 100% into the codebase but does the `trimPaddedZerosFromDecimalPlaces` also properly handle thousand-separators, positve/negative-signs, limited-decimal-places etc?

Reply:
```js
/**
     * Remove the trailing zeros in the decimal part of a number.
     *
     * @param {string} numericString
     * @returns {*}
     */
    static trimPaddedZerosFromDecimalPlaces(numericString) {
        numericString = String(numericString);
        if (numericString === '') {
            return '';
        }

        const [integerPart, decimalPart] = numericString.split('.');
        if (this.isUndefinedOrNullOrEmpty(decimalPart)) {
            return integerPart;
        }

        const trimmedDecimalPart = decimalPart.replace(/0+$/g, '');

        let result;
        if (trimmedDecimalPart === '') {
            result = integerPart;
        } else {
            result = `${integerPart}.${trimmedDecimalPart}`;
        }

        return result;
    }
```